boards/ruiside/art_pi2/doc/index.rst

```diff
@@ -0,0 +1,252 @@
+  - 2x AES coprocessors including one with DPA resistance
+  - Public key accelerator, DPA resistant
+  - On-the-fly decryption of Octo-SPI external memories
+  - HASH hardware accelerator
+  - True random number generator, NIST SP800-90B compliant
+  - 96-bit unique ID
+  - Active tampers
+  - True Random Number Generator (RNG) NIST SP800-90B compliant
+
+- Clock management:
+
+  - 24 MHz crystal oscillator (HSE)
+  - 32768 Hz crystal oscillator for RTC (LSE)
+  - Internal 64 MHz (HSI) trimmable by software
+  - Internal low-power 32 kHz RC (LSI)( |plusminus| 5%)
+  - Internal 4 MHz oscillator (CSI), trimmable by software
+  - Internal 48 MHz (HSI48) with recovery system
+  - 3 PLLs for system clock, USB, audio, ADC
+
+- Power management
+
+  - Embedded regulator (LDO) with three configurable range output to supply the digital circuitry
+  - Embedded SMPS step-down converter
+
+- RTC with HW calendar, alarms and calibration
+- Up to 152 fast I/Os, most 5 V-tolerant, up to 10 I/Os with independent supply down to 1.08 V
+- Up to 16 timers and 2 watchdogs
+
+  - 16x 16-bit
+  - 4x 32-bit timers with up to 4 IC/OC/PWM or pulse counter and quadrature (incremental) encoder input
+  - 5x 16-bit low-power 16-bit timers (available in Stop mode)
+  - 2x watchdogs
+  - 1x SysTick timer
+
+- Memories
+
+  - Up to 64KB Flash, 2 banks read-while-write
+  - 1 Kbyte OTP (one-time programmable)
+  - 640 KB of SRAM including 64 KB with hardware parity check and 320 Kbytes with flexible ECC
+  - 4 Kbytes of backup SRAM available in the lowest power modes
+  - Flexible external memory controller with up to 16-bit data bus: SRAM, PSRAM, FRAM, SDRAM/LPSDR SDRAM, NOR/NAND memories
+  - 2x OCTOSPI memory interface with on-the-fly decryption and support for serial PSRAM/NAND/NOR, Hyper RAM/Flash frame formats
+  - 1x HEXASPI memory interface with on-the-fly decryption and support for serial PSRAM/NAND/NOR, Hyper RAM/Flash frame formats
+  - 2x SD/SDIO/MMC interfaces
+
+- Rich analog peripherals (independent supply)
+
+  - 2x 12-bit ADC with up to 5 MSPS in 12-bit
+  - 1x Digital temperature sensor
+
+- 35x communication interfaces
+
+  - 1x USB Type-C / USB power-delivery controller
+  - 1x USB OTG full-speed with PHY
+  - 1x USB OTG high-speed with PHY
+  - 3x I2C FM+ interfaces (SMBus/PMBus)
+  - 1x I3C interface
+  - 7x U(S)ARTS (ISO7816 interface, LIN, IrDA, modem control)
+  - 2x LP UART
+  - 6x SPIs including 3 muxed with full-duplex I2S
+  - 2x SAI
+  - 2x FDCAN
+  - 2x SD/SDIO/MMC interface
+  - 2x 16 channel DMA controllers
+  - 1x 8- to 16- bit camera interface
+  - 1x HDMI-CEC
+  - 1x Ethernel MAC interface with DMA controller
+  - 1x 16-bit parallel slave synchronous-interface
+  - 1x SPDIF-IN interface
+  - 1x MDIO slave interface
+
+- CORDIC for trigonometric functions acceleration
+- FMAC (filter mathematical accelerator)
+- CRC calculation unit
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
```

Comment:
This is taking 2 pages on the documentation page, I would just remove it. People should click on "More information..." links if they need more info, and doc page shouldn't be unnecessarily cluttered by these.

```suggestion
```